toolchain/crosstool-ng/crosstool.config

```diff
@@ -399,12 +414,14 @@ CT_BINUTILS_V_2_36=y
 # CT_BINUTILS_V_2_28 is not set
 # CT_BINUTILS_V_2_27 is not set
 # CT_BINUTILS_V_2_26 is not set
-CT_BINUTILS_VERSION="2.36.1"
+CT_BINUTILS_VERSION="2.35.1"
```

Comment:
Downgraded binutils just to be safe when debugged the issues with compilation and forgot to upgrade back. But I guess it's better to have binutils matching glibc?

Reply:
It should match what is in the official toolchain ideally. Which would match what is on device if it ends up on device. I don't really think we want to downgrade them unless they actually need to be downgraded.

Reply:
Still haven't received official toolchain for 3.6, sadly

Nothing from binutils end up on device, as far as I can tell. Also, 3.1.15 toolchain included 2.34.0 binutils, not 2.36.1:
```
# /opt/codex/rm11x/3.1.15/sysroots/x86_64-codexsdk-linux/usr/libexec/arm-remarkable-linux-gnueabi/gcc/arm-remarkable-linux-gnueabi/9.3.0/ld --version
GNU ld (GNU Binutils) 2.34.0.20200220
```

I don't think that binutils versions are that important for linking binaries, and probably we can use either one until we get official toolchain, your call.

Reply:
It looks like you have the latest toolchain now. Could you also open a PR against https://github.com/Eeems-Org/remarkable.guide/ to add the toolchain to https://github.com/Eeems-Org/remarkable.guide/blob/main/src/devel/toolchains.rst with ipfs links?

Reply:
Official toolchain includes 2.38, apparently

```
$ /opt/codex/rm2/4.0.117-1/sysroots/x86_64-codexsdk-linux/usr/libexec/arm-remarkable-linux-gnueabi/gcc/arm-remarkable-linux-gnueabi/11.3.0/ld --version
GNU ld (GNU Binutils) 2.38.20220708
```

Probably also should bump gcc to 11.3.0